Makefile

```diff
@@ -58,15 +58,16 @@ lint:  ## Lint go source code
 
 # --- Protos ---------------------------------------------------------------------
 
-protos:  ## Generate Go pb and grpc binding for .proto files
+proto:  ## Generate Go pb and grpc binding for .proto files
+	$(eval GO_MODULE := $(shell go list -m))
 	protoc \
 		-I proto -I proto/vendor \
-		--go_out=. --go_opt=module=$(shell go list -m) \
-		--go-grpc_out=. --go-grpc_opt=module=$(shell go list -m) \
-		$(shell find proto -name vendor -prune -o -name '*.proto' -print)
+		--go_out=. --go_opt=module=$(GO_MODULE) \
+		--go-grpc_out=. --go-grpc_opt=module=$(GO_MODULE) \
+		$(shell find proto -name vendor -prune -o -regex '.*/[^_].*\.proto' -print)
```

Comment:
Two things - it is what Go does so I figure it makes sense to do the same when not explicitly specifying files, and secondly because I have some other changes lined up and I didn't want those files to be compiled, so I just renamed them with an underscore.

I will update the commit message with the "why". Thanks for asking.